Libraries/Components/View/ViewPropTypes.js

```diff
@@ -393,29 +393,95 @@ type IOSViewProps = $ReadOnly<{|
    * See https://reactnative.dev/docs/view#accessibilityElementsHidden
    */
   accessibilityElementsHidden?: ?boolean,
+
   /**
-   * Reverses the direction of scroll. Uses native inversion on macOS and scale transforms of -1 elsewhere
+   * Whether this `View` should be rendered as a bitmap before compositing.
+   *
+   * @platform ios
+   *
+   * See https://reactnative.dev/docs/view#shouldrasterizeios
    */
-  inverted?: ?boolean, // TODO(macOS GH#774)
+  shouldRasterizeIOS?: ?boolean,
+|}>;
 
-  onDoubleClick?: ?(event: SyntheticEvent<{}>) => mixed, // TODO(macOS GH#774)
+// [TODO(macOS GH#774)
+type MacOSViewProps = $ReadOnly<{|
+  /**
+   * Fired when a dragged element enters a valid drop target
+   *
+   * @platform macos
+   */
+  onDragEnter?: (event: MouseEvent) => void,
 
   /**
-   * When `accessible` is true, the system will try to invoke this function
-   * when the user performs accessibility tap gesture.
+   * Fired when a dragged element leaves a valid drop target
    *
-   * See http://facebook.github.io/react-native/docs/view.html#onaccessibilitytap
+   * @platform macos
    */
-  onAccessibilityTap?: ?() => void,
+  onDragLeave?: (event: MouseEvent) => void,
 
   /**
-   * When `accessible` is `true`, the system will invoke this function when the
-   * user performs the magic tap gesture.
+   * Fired when an element is dropped on a valid drop target
    *
-   * See https://reactnative.dev/docs/view#shouldrasterizeios
+   * @platform macos
    */
-  shouldRasterizeIOS?: ?boolean,
+  onDrop?: (event: MouseEvent) => void,
+
+  /**
+   * Specifies the Tooltip for the view
+   * @platform macos
+   */
+  tooltip?: ?string,
+
+  /**
+   * Specifies whether the view should receive the mouse down event when the
+   * containing window is in the background.
+   *
+   * @platform macos
```

Comment:
Some props have `@platform macos` and some do not. Is that intentional?

Reply:
Good catch! They probably all should have it. Also other props that were Android or IOS only that we support on macOS (focusable comes to mind). I think I'll _just_ update focusable because I don't want to go through all the iOS props.